IBPSA/Resources/Scripts/Dymola/Airflow/Multizone/Examples/OneOpenDoor.mos

```diff
@@ -1,4 +1,4 @@
-simulateModel("IBPSA.Airflow.Multizone.Examples.OneOpenDoor", stopTime=7200, method="CVode", tolerance=1e-06, resultFile="OneOpenDoor");
+simulateModel("IBPSA.Airflow.Multizone.Examples.OneOpenDoor", stopTime=7200, numberOfIntervals=120, method="CVode", tolerance=1e-09, resultFile="OneOpenDoor");
```

Comment:
I don't think any of these changes are needed. Also, I think this is the first model where we use such high a tolerance, which is surprising for such simple a model. Can you revert this?